server.ts

```diff
@@ -42,13 +46,10 @@ app.use( function( req, res ) {
 } );
 
 // Start listening on the service port
-app.listen( port, host, function() {
+app.listen( +port, host, function() {
```

Comment:
What does this plus sign indicate? Thanks!

Reply:
It's just casting the string to a number, I can also change it to `parseInt(port)` if you think that's more readable.

This line was changed in that commit because changing the way modules were imported provided the TypeScript transpiler with type information for `app.listen`, revealing that it expects that parameter to be a number.

Reply:
Ah okay, yes please I think thats more readable. I didn't realize we could apply that in the arguments, neat!

Reply:
It would actually need to be changed to `typeof port === "number" ? port : parseInt ( port )` since `port` can be either a string or a number, and `parseInt` only accepts a string. Should I still change it, or is that even harder to read?

Reply:
Disregard that, I can just use `parseInt` on line 11 to make it cleaner

Reply:
Good point, just leave it as a plus. We can put a comment port is being typecasted with the plus if we want.